charts/spire/charts/spire-server/templates/tests/test-tornjak-connection.yaml

```diff
@@ -14,13 +14,7 @@ spec:
     - name: wget-tornjak-backend
       image: busybox
       command: ['wget']
-      args: ['--no-check-certificate', '-O', '/dev/null', 'http://{{ include "spire-tornjak.backend" . }}:10000/api/tornjak/serverinfo']
-      securityContext:
-        {{- toYaml .Values.securityContext | nindent 8 }}
-    - name: wget-tornjak-frontend
-      image: busybox
-      command: ['wget']
-      args: ['--no-check-certificate', '-O', '/dev/null', 'http://{{ include "spire-tornjak.frontend" . }}:3000']
+      args: ['--no-check-certificate', '-O', '/dev/null', 'http://{{ include "spire-tornjak.backend" . }}:10000/']
```

Comment:
Please use the bash chainguard image to this is aligned with #175 which is ready to merge. Also see below for the suggestion. (can't do the suggestion here somehow.